fetch/main.js

```diff
@@ -0,0 +1,14 @@
+async function logUsers() {
+  try {
+    const response = await fetch('https://jsonplaceholder.typicode.com/users');
+    const result = await response.json();
+    console.log(result);
+    if (response.ok === false) {
```

Comment:
You have to check response.ok _before_ calling response.json(). If the response is not OK then there may not be a JSON response at all.